keyboards/planck/keymaps/jcorrado/keymap.c

```diff
@@ -0,0 +1,116 @@
+/* Copyright 2015-2017 Jack Humbert
+ * Copyright 2018 Jereme Corrado
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "planck.h"
+#include "action_layer.h"
+
+enum {
+  _QWERTY = 0,
+  _NUM,
+  _SYM_L,
+  _SYM_R
+};
+
+#define LMOD_0 LT(_SYM_L, KC_TAB)  /* Tab, hold: Sym layer */
+#define LMOD_1 LT(_NUM, KC_ESC)    /* Esc, hold: Num layer */
+#define LMOD_2 LSFT_T(KC_CAPS)     /* Caps Lock, hold: Left Shift */
+
+#define RMOD_0 LT(_SYM_R, KC_BSPC) /* Backspace, hold: Sym layer */
+#define RMOD_1 LT(_NUM, KC_QUOT)   /* Quote, hold: Nu layer */
+#define RMOD_2 RSFT_T(KC_ENT)      /* Enter, hold: Right Shift */
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/*
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | Tab/   | Q      | W      | E      | R      | T      | Y      | U      | I      | O      | P      | BacSp/ |
+ * | Sym    |        |        |        |        |        |        |        |        |        |        | Sym    |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | Esc/   | A      | S      | D      | F      | G      | H      | J      | K      | L      | :      | "      |
+ * | Num    |        |        |        |        |        |        |        |        |        | ;      | '/Num  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | CapLk/ | Z      | X      | C      | V      | B      | N      | M      | <      | >      | ?      | Shift/ |
+ * | Shift  |        |        |        |        |        |        |        | ,      | .      | /      | Enter  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ * | Toggle | ~      | {pipe} | Alt    | Ctrl   | Space  | Space  | Ctrl   | Alt    | {      | }      |        |
+ * | Num    | `      | \      |        |        |        |        |        |        | [      | ]      |        |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+ */
+
+[_QWERTY] = {
```

Comment:
All your layers should be formatted to use an appropriate LAYOUT macro, as seen here:

https://github.com/qmk/qmk_firmware/blob/27e216fc1a481bb3e65832458bd2b199cb499e0b/keyboards/planck/keymaps/default/keymap.c#L58-L63